src/Processing/DefaultProcessor.php

```diff
@@ -527,7 +534,7 @@ protected function handleClassCallbacks(
 				$type->addError($catchedType);
 			}
 
-			throw InvalidData::create($type);
+			throw InvalidData::create($type, $data);
```

Comment:
I think there should also be `NoValue` in case `$catchedType === $type` because user is supposed to add error to original type himself, but I have to find out myself